d3/hyper.js

```diff
@@ -58,6 +58,24 @@ function convertHyper(node, parentKey) {
             }
         });
 
+        // Add the following keys as children only when they refer to objects
+        var objectKeys = ["source"];
```

Comment:
why can't we simply add `source` to the `childKeys` array?

Reply:
Both the `source` key and the `value` key (already in `childKeys`) are used for input objects and simple values, e.g., aggregations. There was no way to preserve existing behavior without special case handling of keys by name inside the `childKeys` iterator. Otherwise, all instances of `source` would be treated as children, adding additional verbosity.